R/lehdr.R

```diff
@@ -31,14 +31,17 @@
 #' @importFrom httr GET stop_for_status HEAD write_disk
 #' @importFrom glue glue
 #' @importFrom stats na.omit
+#' @importFrom stringr str_sub
 #'  
 #' @export
 
-grab_lodes <- function(state, year, lodes_type = c("od", "rac", "wac"), 
+grab_lodes <- function(state, year, 
+                       lodes_type = "od", 
                        job_type = c("JT00", "JT01", "JT02", "JT03", "JT04", "JT05"), 
                        segment = c("S000", "SA01", "SA02", "SA03", "SE01", "SE02",
                                    "SE03", "SI01", "SI02", "SI03"),  
-                       tract = FALSE, state_part = NULL, 
+                       agg_geo = "block",
```

Comment:
this is the biggest functional change -- tract is no longer an argument, and instead you'd need to set `agg_geo = "tract"`

Reply:
Looking at the code further down this pretty damn slick. Can we get a better name for the parameter and add state? I know it hardly matters because it's two digits but might as well put the whole hierarchy in there.

Reply:
Yep - let me know what you want for a name and I can make it happen? Might be easiest In a second pull request? If you can approve this, then I can make changes to the name, vignette and README.